src/cjs/session.ts

```diff
@@ -0,0 +1,30 @@
+import { remote } from 'webdriverio';
+
+import { CJSElectronLauncher, CJSElectronService } from './classes.js';
+import { CUSTOM_CAPABILITY_NAME } from './constants.js';
+import { ElectronServiceOptions } from './types.js';
+import { Options } from '@wdio/types';
+
+export async function init(opts: ElectronServiceOptions) {
```

Comment:
How about dynamically importing the ESM module here to avoid duplication?

Reply:
It causes problems with rollup on the `example` E2Es: https://github.com/webdriverio-community/wdio-electron-service/actions/runs/8908865877/job/24465227351

I'll see if I can remove the bundling - we're not bundling the `example-cjs` app but the other two proved difficult.

Reply:
I removed rollup for the `example` (ESM) main, trying different things for preload - which needs to be CJS.  Now getting "exports is not defined in ES module scope" for the service as imported by `@wdio/utils` safeImport.  This happens regardless of bundling or lack thereof in `example`, so it must be a result of the dynamic import here.

Reply:
I kept the bundling for preload scripts in the ESM examples - converting them to CJS syntax didn't help.  I can't see a way of removing the duplication here with `session.ts`.

Reply:
ok, we can keep it this way but let's add a comment documenting why the code is duplicated.